### ISSUE TITLE
Review annotations wrt GO:0060070 'canonical Wnt signaling pathway'

### ISSUE THREAD

**Issue by vanaukenk** [geneontology/go-annotation#1966]
As part of the signaling workshop, Helen and I have compiled a list of annotations to review wrt GO:0060070 'canonical Wnt signaling pathway'.

There is a Google spreadsheet that lists 8 different sheets for the genes that we believe are accepted as core members of the 'canonical Wnt signaling pathway' plus one additional sheet for genes annotated to 'canonical Wnt signaling pathway' that are not core members, i.e. are downstream targets, function upstream to modify Wnt ligands prior to secretion, etc.

https://docs.google.com/spreadsheets/d/1_OaSa5PROAS9pRYqYtmE5WDt50CSyZkekI2cI7NUy5s/edit#gid=504445784

If curators have any questions about the suggested actions, we can discuss on this thread and/or an annotation call.

As always, please remove yourself from the assignees list when you've finished.

Thanks in advance for reviewing these annotations.

@vanaukenk
@ggeorghiou 
@BarbaraCzub 
@RLovering 
@ukemi 
@hdrabkin 
@sabrinatoro 
@jinhuiz
@slaulederkind 
@sandyl27

**Comment by BarbaraCzub** [comment]
Hi Kimberly, 
I've changed the ARUK-UCL UniProtKB:P35222 annotation, as you suggested, but I am not able to edit the spreadsheet (the spreadsheet mode is displayed as 'View only' on my screen).
Barbara

**Comment by slaulederkind** [comment]
RGD annotation removed.

**Comment by sabrinatoro** [comment]
Going through the annotations... not done yet but I have questions/comment:
Questions: 
- are wnt ligands considered part of the wnt canonical pathway? (just want to confirm)
- what should we do with the "non-core2canonical" annotations? Should we delete them? Should we use Noctua and make "downstream of Wnt pathway" (or whatever the relation is... I haven't looked into it carefully) annotations? 

Comment: 
- for the ZFIN annotation to tcf7l2: the author of the publication did not show txf7l2 as part of the wnt pathway. This involvement is given as an "author traceable statement". I removed the annotation from ZFIN and used Noctua to annotate the gene in accordance to what was reported in the publication. 
(I didn't know where to add this note in the spreadsheet)

@vanaukenk

**Comment by hattrill** [comment]
Hi @BarbaraCzub, sorry missed your comment. Have made the sheet editable by anyone via link and have added you! (belt and braces, so to speak).

**Comment by hattrill** [comment]
Hi @sabrinatoro and all:
Here's a link to the wnt GOC presentation:
https://docs.google.com/presentation/d/1rjPihu4doOrffCUzY6UbEzv4oqvwNV2Z5gUR8OLmLKo/edit#slide=id.g39ecfacf9d_2_377

Wnts are part of the canonical pathway, but they are also part of the non-canonical pathway. There are a few examples of Wnt ligands that actually block the pathway, so would be negative regulators. If you can review your wnt ligands, then so much the better. From a review perspective there were just too many to find a sensible way to peg them down.

**Comment by hattrill** [comment]
@sabrinatoro For the "non-core" ones, they are probably a mixed bag of pathway regulators, downstream or upstream so each require a bit of digging to find the correct way to re-house them. With the new qualifiers, the upstream annotations might be best captured as this.

With the FlyBase ones, I have put all ones that are involved in Wnt ligand biogenesis/secretion under "Wnt protein secretion", so I don't lose them and can review them in the light of the new qualifiers.

For downstream components, that have their expression changed by the pathway, I have removed the annotations. The expression could be captured in a noctua model or via an annotation extension for beta-catenin regulation of transcription.


I think that for tcf7l2, could mark as "changed" in the spreadsheet.

**Comment by sandyl27** [comment]
So I have a paper "IQGAP1 Functions as a Modulator of Dishevelled Nuclear Localization in Wnt Signaling" and in the presentation in NY I don't remember DVL localizing to the nucleus so I was wondering if someone could help me with this?

PMID:23577172

I was thinking GO:0006606 (Figure 3) and GO:0045893 (Figure 4)

**Comment by hattrill** [comment]
@sandyl27 the role of dvl in the nucleus is a bit contentious, as opposed to its cytoplasmic role as a signaling scaffold in canonical signaling - PMID:26969973

I think that for this paper, IQGAP1: GO:0006606 seems appropriate; I would be reluctant to add GO:0045893 as I don't see direct evidence for this.

**Comment by sandyl27** [comment]
Okay! thank you!

**Comment by RLovering** [comment]
2 annotations remain, new GO terms required before these can be updated. I haven't yet requested one of the terms in case this is too specific (see table)
Ruth

**Comment by hattrill** [comment]
Yes, looks like the LPP3-PTEN is a separate pathway, even more divergent than Norrin.

@vanaukenk - I wonder that if we have many more examples like this, that it would justify a parent kinda-like "receptor-mediated signaling pathway via beta catenin"(....but better)

![screen shot 2018-06-18 at 08 19 46](https://user-images.githubusercontent.com/11865746/41523473-ff45e966-72d1-11e8-8a89-35d0feafc745.png)

**Comment by RLovering** [comment]
Hi Helen

I was just thinking of suggesting something like 'receptor signaling pathway via beta-catenin' too. 

Also this would follow what we have decided to do with JAK-STAT: receptor signaling pathway via JAK-STAT

However, I have opened a new ticket for this as I can see this will be distracting on this ticket #15914

Best

Ruth

**Comment by RLovering** [comment]
I have completed all the Parkinson's and BHF annotations that I can. I have created an additional tab for annotations that require new GO terms before they can be revised.
Thanks
Ruth

**Comment by hattrill** [comment]
Ace!

**Comment by BarbaraCzub** [comment]
I've fixed the ARUK-UCL annotations.

**Comment by hattrill** [comment]
Ace! Thanks.

**Comment by pgarmiri** [comment]
All UniProt EBI annotations have been reviewed and updated accordingly. SIB curators have been contacted.

**Comment by hattrill** [comment]
Lovely, thanks!

**Comment by RLovering** [comment]
ucl done

**Comment by hdrabkin** [comment]
MGI finished
